parties/calendar_feeds.py

```diff
@@ -0,0 +1,96 @@
+# This file maintains the calendar feeds cache, and provides feed data to the 
+# calendar feed endpoints
+
+from django.urls import reverse
+import time
+from datetime import date, timedelta
+import os
+from os import path
+import icalendar
+from parties.models import (
+    Competition,
+    Organiser,
+    Party,
+    PartyExternalLink,
+    PartySeries,
+    PartySeriesExternalLink,
+)
+
+CACHE_DIR='cache/demozoo/calendar_feeds/'
```

Comment:
We have redis set up for caching, although we're currently only using it for maintenance reports (https://github.com/demozoo/demozoo/blob/main/maintenance/reports.py) - could we use this rather than the filesystem?